test/server/cards/19.1-AS01/Neutral/ToShowThePath.spec.js

```diff
@@ -122,7 +124,9 @@ describe('To Show the Path', function () {
                 this.player2.pass();
                 this.player1.clickCard(this.toShowThePath);
                 this.player1.clickCard(this.motoYouth);
-                expect(this.getChatLogs(3)).toContain('player1 plays To Show the Path to make targeting Moto Youth and Duelist Training with any card ability by opponents cost 1 more fate');
+                expect(this.getChatLogs(3)).toContain(
+                    'player1 plays To Show the Path to make targeting Moto Youth and Duelist Training with any card ability by opponents cost 1 more fate'
+                );
```

Comment:
This is a misleading message because it makes it sound like it's not going to apply to future cards.

ex: `player1 plays To Show the Path to make player2 pay 1 fate whenever they target Moto Youth or its attachments with a card ability`